rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -44,6 +49,40 @@ object Runtime {
         )
     }
 
+  class ErrorLog() extends FunctorTell[Task, InterpreterError] {
```

Comment:
Not strictly necessary to have the empty parens.  Are you trying to convey the stateful nature of instances?